src/pform.ml

```diff
@@ -152,6 +154,8 @@ module Map = struct
       ; "path-no-dep", deleted_in ~version:(1, 0) Macro.Path_no_dep
       ; "ocaml-config", macro Ocaml_config
       ; "env", since ~version:(1, 4) Macro.Env
+      ; "var", since ~version:(1, 6) Macro.Config_var (* XXX check extension, or
```

Comment:
Is this possible to have a "since" on an extension syntax? it seems not, so I think this would have to live on the main syntax.

Reply:
It supported in the parser, but maybe not on pforms yet.